### ISSUE TITLE
Correction needed - UNIR (Porto)

### ISSUE THREAD

**Issue by PedroMTG13** [osmlab/name-suggestion-index#9342]
Hey!

Unir operator wikidata (https://nsi.guide/?t=transit&k=highway&v=bus_stop#unir-d62b2b  , https://www.wikidata.org/wiki/Q124098736) could have better tags. Is needed to change the now network to Operator, and put 'Andante' as the new network.

Unir is the one operator for Porto metropolitan area, as is STCP (Sociedade de Transportes Colectivos do Porto) and Metro do Porto (subway).
Theses 3 operators are oblied to use Andante card systems, and tarif zones. 

In Porto's case Andante is network all modes are connected, 
and each of the companies are the operators.

subway - Metro do Porto
inner city - STCP
suburbs and metropolitan area - UNIR

I think STCP and Metro do Porto dont have wikidata. Could be created also

**Comment by kjonosm** [comment]
- Metro do Porto, https://www.wikidata.org/wiki/Q29757
- UNIR, https://www.wikidata.org/wiki/Q124098736
- STCP, https://www.wikidata.org/wiki/Q366110

**Comment by Snowysauce** [comment]
Andante is apparently already in the NSI (https://nsi.guide/?t=transit&k=highway&v=bus_stop#andante-d62b2b). It needs a Wikidata entry to appear in OSM editors as a suggestion/preset ([see here for details](https://github.com/osmlab/name-suggestion-index/wiki/Technical-Details#availability-of-entries-in-downstream-projects)).

**Comment by Snowysauce** [comment]
A quick search of Wikidata returns [Andante ticket](https://www.wikidata.org/wiki/Q4753591) (Q4753591). @PedroMTG13, is this the same "Andante" network you mentioned?

**Comment by PedroMTG13** [comment]
@Snowysauce Exactly, the Andante network is the same for trains trams buses and metro in porto metro area. But each of these is owned by different companies

**Comment by Snowysauce** [comment]
I've added the linked Wikidata ID to the existing Andante entry in `transit/route/bus`. I left the operator unset because it's not clear to me whether the bus lines have one operator or multiple operators.

Also, I noticed that "Andante" is listed under `matchNames` for the Metro do Porto entry in `transit/route/light_rail`. Should that entry actually have `network=Andante`, `operator=Metro do Porto`?

**Comment by PedroMTG13** [comment]
Thanks
